packages/react-components/src/components/Tag/Tag.stories.tsx

```diff
@@ -19,6 +21,29 @@ const exampleAvatar = (
     data-testid="lc-tag-avatar"
   />
 );
+const kinds: Array<TagKind> = [
+  'default',
+  'info',
+  'warning',
+  'success',
+  'error',
+  'purple',
+  'black',
+  'gray',
+  'gradient01',
+  'gradient02',
+  'gradient03',
+  'gradient04',
+  'gradient05',
+  'gradient06',
+  'gradient07',
+  'gradient08',
+  'gradient09',
+  'gradient10',
+  'gradient11',
+];
```

Comment:
I think we can move it to constants and then, based on this, we can generate our type `TagKind`. Thanks to this we will always have the stories synchronized automatically.